am-lib/src/main/java/uk/gov/hmcts/reform/amlib/repositories/AccessManagementRepository.java

```diff
@@ -32,7 +32,8 @@
         + "and access_management.service_name = :serviceName "
         + "and access_management.resource_type = :resourceType "
         + "and access_management.resource_name = :resourceName "
-        + "and access_management.attribute = :attributeAsString")
+        + "and access_management.attribute = :attributeAsString "
+        + "or access_management.attribute like concat(:attributeAsString, '/','%')")
```

Comment:
There is still missing whitespace between args of concat.

Reply:
Sorry didnt realise you wanted whitespace here changed now.